certbot-apache/certbot_apache/tests/parsernode_test.py

```diff
@@ -8,28 +8,41 @@
 
 
 
-class DummyCommentNode(interfaces.CommentNode):
+class DummyCommentNode(interfaces.ParserNode):
     """ A dummy class implementing CommentNode interface """
     ancestor = None
     comment = ""
     dirty = False
+    filename = ""
```

Comment:
Can we define these attributes in `__init__` for this and all other classes in this file?

Reply:
We can, do you think we should do that in context of this PR, or the VirtualHost creation one?

Reply:
I have a slight preference for this one since it's the one making the change that these no longer need to be class variables, but I don't really care.